pkg/orchestration/wiring/entangler.go

```diff
@@ -158,39 +156,17 @@ func (en *Entangler) Entangle(state *orch_v1.State, context *EntangleContext) En
 		Label:   context.Label,
 	}
 
-	// Atlassian Specific Things
-	legacyConfigFunc := en.GetLegacyConfigFunc
-	if legacyConfigFunc == nil {
-		return &EntangleResultFailure{
-			Error: errors.New("missing legacy config"),
-		}
-	}
-	legacyConfig := legacyConfigFunc(location)
-	if legacyConfig == nil {
-		return &EntangleResultFailure{
-			Error: errors.Errorf("no legacy config for %s", location),
-		}
-	}
-
 	tags := make(map[voyager.Tag]string)
 	for tag, val := range context.ServiceProperties.UserTags {
 		tags[tag] = val
 	}
 
-	tags[en.TagNames.ServiceNameTag] = string(context.ServiceName)
-	tags[en.TagNames.BusinessUnitTag] = context.ServiceProperties.BusinessUnit
-	tags[en.TagNames.ResourceOwnerTag] = context.ServiceProperties.ResourceOwner
-	tags[en.TagNames.EnvironmentTypeTag] = string(location.EnvType)
-	tags[en.TagNames.PlatformTag] = "voyager"
-	tags[legacyEnvironmentTagName] = legacyConfig.MicrosEnv
```

Comment:
This lookup is what requires the tags generation to be a function and not just a static map. The other keys could be seen as opinionated defaults.